spec/function.dd

```diff
@@ -872,15 +872,12 @@ void main()
 
 $(H4 $(LNAME2 inline-functions, Inline Functions))
 
-        $(P There is no inline keyword. The compiler makes the decision whether to
-        inline a function or not, analogously to the register keyword no
-        longer being relevant to a
-        compiler's decisions on enregistering variables.
-        (There is no register keyword either.))
-
+        $(P The compiler makes the decision whether to inline a function or not. 
+        This decision may be controlled by $(LINK2 pragma.html#inline, `pragma(inline)`),
+        assuming that the compiler implements it, which is not mandatory.)
+        
         $(P If a $(GLINK2 expression, FunctionLiteral) is immediately called,
-        its inlining would be enforced normally.)
-
+        its inlining would be enforced normally.)        
```

Comment:
What does "would be enforced normally" actually mean? I suppose, "users can expect the call to be inlined"? Please change it accordingly, if this is the intended meaning.